text/0119-instance-options.md

```diff
@@ -0,0 +1,459 @@
+**Expected value**: an integer
+
+Sets the number of jobs available during the indexation.
+
+By default, in machines with multi-core processors, the indexer avoids using more than half of the available processing units. For example, if your machine has twelve cores, the indexer will try to use six of them at most. This ensures Meilisearch is always ready to perform searches, even while you are updating an index.
+
+Obviously, multi-threading is not possible in machines with only one processor core.
+
+If the number set is higher than the real number of core available in the machine, Meilisearch will use the maximum number of available cores.
+
+#### SSL authentication path
+
+**Environment variable**: `MEILI_SSL_AUTH_PATH`
+**CLI option**: `--ssl-auth-path`
+**Default value**: `None`
+**Expected value**: a filepath
+
+Enables client authentication in the specified path.
+
+#### SSL certificates path
+
+**Environment variable**: `MEILI_SSL_CERT_PATH`
+**CLI option**: `--ssl-cert-path`
+**Default value**: `None`
+**Expected value**: a filepath pointing to a valid SSL certificate
+
+Sets the server's SSL certificates.
+
+Value must be a path to PEM-formatted certificates. The first certificate should certify the KEYFILE supplied by `--ssl-key-path`. The last certificate should be a root CA.
+
+#### SSL key path
+
+**Environment variable**: `MEILI_SSL_KEY_PATH`
+**CLI option**: `--ssl-key-path`
+**Default value**: `None`
+**Expected value**: a filepath pointing to a valid SSL keyfile
+
+Sets the server's SSL keyfiles.
+
+Value must be a path to an RSA private key or PKCS8-encoded private key, both in PEM format.
+
+#### SSL OCSP path
+
+**Environment variable**: `MEILI_SSL_OCSP_PATH`
+**CLI option**: `--ssl-ocsp-path`
+**Default value**: `None`
+**Expected value**: a filepath pointing to a valid OCSP certificate
+
+Sets the server's OCSP file. *Optional*
+
+Reads DER-encoded OCSP response from OCSPFILE and staple to certificate.
+
+#### SSL require auth
+
+**Environment variable**: `MEILI_SSL_REQUIRE_AUTH`
+**CLI option**: `--ssl-require-auth`
+**Default**: Disabled
+
+⚠️ This command-line option does not take any values. Assigning a value will throw an error.
+
+Makes SSL authentication mandatory.
+
+Sends a fatal alert if the client does not complete client authentication.
+
+#### SSL resumption
+
+**Environment variable**: `MEILI_SSL_RESUMPTION`
+**CLI option**: `--ssl-resumption`
+**Default**: Disabled
+
+⚠️ This command-line option does not take any values. Assigning a value will throw an error.
+
+Activates SSL session resumption.
+
+#### SSL tickets
+
+**Environment variable**: `MEILI_SSL_TICKETS`
+**CLI option**: `--ssl-tickets`
+**Default**: Disabled
+
+⚠️ This command-line option does not take any values. Assigning a value will throw an error.
+
+Activates SSL tickets.
+
+## 4. Technical Aspects
+
+N/A
+
+## 5. Future Possibilities
+
+- Redo the command-line to create a more interactive CLI
```

Comment:
Do we have a autocomplete to the options (like `git` does)?

<img width="313" alt="image" src="https://user-images.githubusercontent.com/4116980/156416391-f0b0c4fc-8fb6-4124-b5bb-28b305b3c39c.png">

Reply:
Not as far as I know, can be a future possibility :)

Reply:
Cool, yeah that's the idea, could be helpful to add in the `Future Possibilities`